[BUG] Unable to enable auto trace route for devices

# If you do not follow this template, your issue may be ignored or delayed due to lack of information.

**Describe the bug**
Unable to enable trace route for devices

**To Reproduce**
Steps to reproduce the behavior:
1. Go to 'Automation'
2. Click on 'Auto Trace Route'
3. Click on a device
4. Click save
5. See error

**Expected behavior**
Saving settings

**Screenshots**
N/A

**Environment (please complete the following information):**
* Host OS: Linux
* Deployment type: Docker
* Database backend: SQLite

**Additional context**
[ERROR] Error updating auto-traceroute node filter: SqliteError: table auto_traceroute_nodes has no column named enabled

    at Database.prepare (/app/node_modules/better-sqlite3/lib/methods/wrappers.js:5:21)

    at BetterSQLiteSession.prepareQuery (file:///app/node_modules/drizzle-orm/better-sqlite3/session.js:23:30)

    at BetterSQLiteSession.prepareOneTimeQuery (file:///app/node_modules/drizzle-orm/sqlite-core/session.js:141:17)

    at QueryPromise._prepare (file:///app/node_modules/drizzle-orm/sqlite-core/query-builders/insert.js:146:81)

    at QueryPromise.run (file:///app/node_modules/drizzle-orm/sqlite-core/query-builders/insert.js:162:17)

    at QueryPromise.execute (file:///app/node_modules/drizzle-orm/sqlite-core/query-builders/insert.js:174:54)

    at QueryPromise.then (file:///app/node_modules/drizzle-orm/query-promise.js:21:17)

    at process.processTicksAndRejections (node:internal/process/task_queues:103:5) {

  code: 'SQLITE_ERROR'

}

--- a/src/services/database.ts
+++ b/src/services/database.ts
@@ -2125,14 +2125,10 @@ class DatabaseService {
   private runAutoTracerouteEnabledMigration(): void {
     try {
       const migrationKey = 'migration_060_auto_traceroute_enabled';
-      const migrationCompleted = this.getSetting(migrationKey);
-
-      if (migrationCompleted === 'completed') {
-        logger.debug('✅ Auto traceroute enabled column migration already completed');
-        return;
-      }
-
-      logger.debug('Running migration 060: Add enabled column to auto_traceroute_nodes...');
+      // Always run the migration check - it's idempotent and verifies
+      // the column exists regardless of whether it was previously marked complete.
+      // This guards against edge cases where the setting was marked complete
+      // but the column is actually missing (e.g., restored backups).
       autoTracerouteEnabledMigration.up(this.db);
       this.setSetting(migrationKey, 'completed');
       logger.debug('✅ Auto traceroute enabled column migration completed successfully');

